protocol/0082-ETHD-ethereum-data-source.md

```diff
@@ -110,81 +110,79 @@ Select {
+2. Ensure that any ethereum oracle events that were generated during network downtime are correctly processed as soon as the network is restored and operational. This means that any termination or settlement actions that would of occurred during downtime are immediately actioned when network is up and we ensure they are processed in sequenced that they were received by the core polling. (<a name="0082-ETHD-041" href="#0082-ETHD-041">0082-ETHD-041</a>)
 
 ### Snapshots
 
-1. Oracle data sources should be stored on snapshots and should be able to be restored from snapshots (<a name="0082-ETHD-040" href="#0082-ETHD-040">0082-ETHD-040</a>)
-2. Restart a network with an active external data source from snapshot. Ensure the data source is active and either catches up all missed events or starts processing new events based on config (<a name="0082-ETHD-041" href="#0082-ETHD-041">0082-ETHD-041</a>)
+1. Oracle data sources linked to markets should be stored on snapshots and should be able to be restored from snapshots. The states of the oracle data sources should be maintained across any markets where they are linked to ethereum data sources. (<a name="0082-ETHD-042" href="#0082-ETHD-042">0082-ETHD-042</a>)
 
 ### Protocol Upgrade
 
-1. Create / amend an external oracle data source and before it is enacted perform a protocol upgrade. The oracle data source should be enacted at the correct time after the upgrade (<a name="0082-ETHD-042" href="#0082-ETHD-042">0082-ETHD-042</a>)
-2. Create / amend an external oracle data source with enactment time that falls during a protocol upgrade. The oracle data source should be enacted immediately after the network is up after the protocol upgrade (<a name="0082-ETHD-043" href="#0082-ETHD-043">0082-ETHD-043</a>)
-3. Phase 2 - Have a network running g with a mix of internal and external active and inactive oracles. Perform a protocol upgrade. Once the network is up , the state of the various oracles should be the same as before the protocol upgrade and either catch up all missed events or start processing new events based on config (<a name="0082-ETHD-044" href="#0082-ETHD-044">0082-ETHD-044</a>)
-
-### Regression
-
-1. The ethereum oracles feature ONLY changes the way we source the data BUT does not change the way the sourced data is filtered / processed / used by the system . So in theory all existing oracle data sourcing tests should pass. In addition , it should be possible to run the existing tests by swapping the interval oracle data source for an external one (<a name="0082-ETHD-045" href="#0082-ETHD-045">0082-ETHD-045</a>)
+1. Have a network running with a couple of markets with a mix of internal, external, open and ethereum oracles. Perform a protocol upgrade. Once the network is up , the state of the various data sources should be the same as before the protocol upgrade (<a name="0082-ETHD-043" href="#0082-ETHD-043">0082-ETHD-043</a>)
```

Comment:
clarify futures market and add something for perps (perps can come in a later PR if needed)

Reply:
Clarified for futures market and added a similar AC for perpetual markets.